### PR TITLE
html-relative-links - Lamberto Martinez

### PR DESCRIPTION
**Pull request by Lambpato**
https://user-images.githubusercontent.com/34325124/214222754-92e8ffb7-30ed-4559-a7c9-1aac12fcfa46.mov

### CODE REVIEW

**Comment by Lambpato** [comment]
Screenshot 2
<img width=
023-01-23 at 10 03 55 PM" src="https://user-images.githubusercontent.com/34325124/214222893-54188cd7-cb2c-4802-9f93-f7384e9f5709.png">
Screenshot 2023-01-23 at 10 04 13 PM
Screenshot 2023-01-23 at 10 04 27 PM

File: html-relative-links/about-me/about-me.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
```
Comment (JenBowman1) [inline html-relative-links/about-me/about-me.html:14]: Hi Lamberto, when referring to the file itself, you don't need to include the path. For instance, if you are in the about-me.html file, your href attribute would just be "about-me.html".

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
Hi Lamberto! Just a few small changes you can see below

File: html-relative-links/about-me/about-me.html (hunk 1 above)
Comment (Lambpato) [reply html-relative-links/about-me/about-me.html:14]: 🫡

File: html-relative-links/about-me/about-me.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me.html">About Me</a></li>
```
Comment (JenBowman1) [inline html-relative-links/about-me/about-me.html:14]: Hi Timothy, when referring to the file itself, you don't need to include the path. For instance, if you are in the about-me.html file, your href attribute would just be "about-me.html".

File: html-relative-links/contact-us/contact-us.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
+    <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
```
Comment (JenBowman1) [inline html-relative-links/contact-us/contact-us.html:16]: see above

File: html-relative-links/projects/projects.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
```
Comment (JenBowman1) [inline html-relative-links/projects/projects.html:15]: see above

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
Just a few small changes !

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**